vips/vip-192/commands-testnet.ts

```diff
@@ -0,0 +1,292 @@
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "updateMultipliers(address,uint256,uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setStakedAt(address[],uint256[])", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "addMarket(address,uint256,uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setLimit(uint256,uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setMaxLoopsLimit(uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "issue(bool,address[])", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "burn(address)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "togglePause()", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "updateAlpha(uint128,uint128)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "updateMultipliers(address,uint256,uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setStakedAt(address[],uint256[])", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "addMarket(address,uint256,uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setLimit(uint256,uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setMaxLoopsLimit(uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "issue(bool,address[])", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "burn(address)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "togglePause()", CRITICAL_TIMELOCK],
+  },
+  {
+    target: PRIME_LIQUIDITY_PROVIDER,
+    signature: "setPrimeToken(address)",
+    params: [PRIME],
+  },
+  {
+    target: UNITROLLER,
+    signature: "_setPrimeToken(address)",
+    params: [PRIME],
+  },
```

Comment:
We need to invoke `setPrimeToken` also on the `XVS_VAULT_PROXY`. Also in the mainnet VIP